python/src/aiconfig/editor/server/server.py

```diff
@@ -307,7 +311,11 @@ def kill_thread(thread_id: int | None):
                     yield from handle_cancellation()
                     return
 
-                if isinstance(text, str):
+                if isinstance(text, Exception):
+                    yield "["
+                    yield json.dumps({"message": "exception", "output": str(text)})
```

Comment:
The structure of the payload should match the same as what `create_cancellation_payload` does:
```
def create_error_payload():
            aiconfig_json = aiconfig_deep_copy.model_dump(exclude=EXCLUDE_OPTIONS) if aiconfig_deep_copy is not None else None
            return json.dumps({"error": {"message": str(text), "code": 500, "data": aiconfig_json}})
```